docs/index.md

```diff
@@ -1,7 +1,7 @@
-# IBI Transit Data Tools (TRANSIT-data-tools)
+# Arcadis TRANSIT-data-tools
```

Comment:
The correct stylized spelling doesn't have hyphen between "data" and "tools"

```suggestion
# Arcadis TRANSIT-datatools
```

Reply:
I think it does as per: https://www.ibigroup.com/ibi-products/transit-data-tools/

Reply:
Hmm okay, I guess other folks have been using it wrong!